app/uk/gov/hmrc/bindingtariffrulingfrontend/service/RulingService.scala

```diff
@@ -42,7 +45,8 @@ class RulingService @Inject()(repository: RulingRepository, bindingTariffClassif
   }
 
   def refresh(reference: String)(implicit hc: HeaderCarrier): Future[Unit] = {
-    val rulingPair: Future[(Option[Ruling], Option[Ruling])] = for {
+
+    val rulingUpdate: Future[(Option[Ruling], Option[Ruling])] = for {
```

Comment:
Could we use `type` here and define a `Future(ExistingRuling, UpdatedRuling)`

Reply:
if you like it...
we could say:

```
    type ExistingRuling = Option[Ruling]
    type UpdatedRuling = Option[Ruling]
    type RulingUpdate = (ExistingRuling, UpdatedRuling)
```

Reply:
What do you think? Better or worse?

Reply:
Ah you've done it 👍